### ISSUE TITLE
Chrome 50 fails to set answer sdp sent from Bowser

### ISSUE THREAD

**Issue by y-element** [EricssonResearch/openwebrtc#594]
Hi, in my environment Chrome 50 started to show error when executing setRemoteDescription() with SDP sent from bowser, as follows (it doesn't occur on Chrome 49 <-> bowser call):
- DOMException: Failed to set remote answer sdp: Session error code: ERROR_CONTENT. Session error description: Failed to set remote video description send parameters..

I could fix the issue by tweaking sdp, not yet by the patch to owr (I feel Chrome handles sdps too strictly but anyway the issue can be fixed by owr). The problem started to happen since Chrome started to include rtx lines for VP9 etc from M50; owr/webrtc.js opts-out lines for VP9+ properly but it won't remove rtx lines referring them.

SDP given from bowser (in response to offer from chrome 50) causing error:
 m=video 9 UDP/TLS/RTP/SAVPF 100 96 97 98
 c=IN IP4 0.0.0.0
 a=rtcp-mux
 a=recvonly
 a=rtpmap:100 VP8/90000
 a=rtpmap:96 rtx/90000
 a=rtpmap:97 rtx/90000
 a=rtpmap:98 rtx/90000
 a=fmtp:96 apt=100
 a=fmtp:97 apt=101
 a=fmtp:98 apt=116
 a=rtcp-fb:100 nack
 a=rtcp-fb:100 nack pli
 a=rtcp-fb:100 ccm fir

I haven't confirmed yet though (I don't have an environment with openwebrtc built for myself), I guess it can be fixed by changing some lines in webrtc.js, to something like:
`return dp && (!dp.parameters || !payload.parameters || (payload.parameters.packetizationMode == dp.parameters.packetizationMode && payload.parameters.apt == dp.parameters.apt));
`

from:
`return dp && (!dp.parameters || !payload.parameters || payload.parameters.packetizationMode == dp.parameters.packetizationMode);`

**Comment by y-element** [comment]
In addition to above changes I'm modifying offer SDP on bowser side :
sdp = sdp.replace(/a=setup:(passive|active)/igm, "a=setup:actpass")
Otherwise sdp cointains "a=setup" line that chrome doesn't accept, when the offer from bowser started after the first negotiation (offer/answer). I haven't yet found the root cause of this problem in owr codes.

**Comment by stefhak** [comment]
@y-element thanks for reporting and looking into. I'll have a look at this.

**Comment by y-element** [comment]
@stefhak Thank you!, so appreciated.

**Comment by stefhak** [comment]
@y-element I think this was fixed when #598 was landed. Could you verify (note: it has not made it into a Bowser version yet)?

**Comment by y-element** [comment]
@stefhak
Thanks, I don't have owr environment right now but I will try to setup owr
& verify the fix.

**Comment by ultravistor** [comment]
I verified the issue reported here (the chrome50-specific one) was fixed, by removing SDP tweaks. Thanks so much. :)

I am experiencing separate problem by the way, which is, when owr initiates call and chrome joins the call, we will see a lot of warning messages and both sides will eventually get muted remote video stream. (I think no content is exchanged during this moment)

** (<unknown>:43473): WARNING **: 'username' not the same for all remote candidates in the session
** (<unknown>:43473): WARNING **: 'password' not the same for all remote candidates in the session
** (<unknown>:43473): WARNING **: 'username' not the same for all remote candidates in the session
** (<unknown>:43473): WARNING **: 'password' not the same for all remote candidates in the session
** (<unknown>:43473): WARNING **: 'username' not the same for all remote candidates in the session
** (<unknown>:43473): WARNING **: 'password' not the same for all remote candidates in the session
** (<unknown>:43473): WARNING **: 'username' not the same for all remote candidates in the session
** (<unknown>:43473): WARNING **: 'password' not the same for all remote candidates in the session

There are no sdp related error from UA, but on owr/safari side, there are error saying "Failed to load resource: unsupported URL", at which point the peerconnection seems to be lost. (Btw as another typical/strange behavior, onaddstream event will be emitted twice, before and right after this error).
![untitled 3](https://cloud.githubusercontent.com/assets/6234857/14404232/d124def8-feab-11e5-8175-3774ec513cf9.png)

For the moment I can avoid/workaround this error by always establishing the call to the same direction. It would be great to see any helps/advices (including separately creating issue etc). Thanks in advance.
